nvflare/app_opt/flower/flower_job.py

```diff
@@ -64,8 +60,6 @@ def __init__(
             per_msg_timeout (float, optional): Timeout for receiving individual messages. Defaults to 10.0 seconds.
             tx_timeout (float, optional): Timeout for transmitting data. Defaults to 100.0 seconds.
             client_shutdown_timeout (float, optional): Timeout for client shutdown. Defaults to 5.0 seconds.
-            stream_metrics (bool, optional): Whether to stream metrics from Flower client to Flare
-            analytics_receiver (AnalyticsReceiver, optional): the AnalyticsReceiver to use to process received metrics.
             extra_env (dict, optional): optional extra env variables to be passed to Flower client
```

Comment:
change is good, we might need to add some doc somewhere to explain this is a change, maybe changelog should be enough.